src/js/dataEntryPanel.js

```diff
@@ -22,7 +22,8 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             dataEntry: ".floec-ca-dataEntryPanel-dataEntry",
             totalValue: ".floec-ca-dataEntryPanel-totalValue",
             totalPercentage: ".floec-ca-dataEntryPanel-totalPercentage",
-            totalLabel: ".floec-ca-dataEntryPanel-totalLabel"
+            totalLabel: ".floec-ca-dataEntryPanel-totalLabel",
+            resetButton: ".floec-ca-dataEntryPanel-resetButton"
```

Comment:
We probably can name the selector and the css selector too to ```reset``` instead of ```resetButton``` that implies this needs to be a button. The reset essentially can be in any form, such as a link etc.